### DISCUSSION TITLE
When opening a pdf, only the pdf should open without an extra empty home tab

### DISCUSSION THREAD

**Discussion by quadans** [sumatrapdfreader/sumatrapdf#4372] [Q&A]
It annoys me to no end that whenever I open any PDF file, two tabs get opened: one with the actual PDF, and the other one an empty "home" tab with some informations i dont care about. How to disable this?

**Comment by kjk** [comment]
https://www.sumatrapdfreader.org/docs/Advanced-options-settings
https://www.sumatrapdfreader.org/settings/settings#NoHomeTab

**Comment by quadans** [comment]
I searched it before but didnt find it. Now I found it, its `NoHomeTab = true` in advanced settings
